contracts/HATVaults.sol

```diff
@@ -1075,33 +1075,33 @@ contract  HATVaults is Governable, ReentrancyGuard {
     * @param _pid The pool id
     **/
     function emergencyWithdraw(uint256 _pid) external {
-        checkWithdrawAndResetwithdrawEnableStartTime(_pid);
+        checkWithdrawAndResetWithdrawEnableStartTime(_pid);
         _emergencyWithdraw(_pid);
     }
 
-    function getPoolRewardsLevels(uint256 _pid) external view returns(uint256[] memory) {
-        return poolsRewards[_pid].rewardsLevels;
+    function getBountyLevels(uint256 _pid) external view returns(uint256[] memory) {
+        return bountyInfos[_pid].bountyLevels;
     }
 
-    function getPoolRewards(uint256 _pid) external view returns(PoolReward memory) {
-        return poolsRewards[_pid];
+    function getBountyInfo(uint256 _pid) external view returns(BountyInfo memory) {
+        return bountyInfos[_pid];
     }
 
     // GET INFO for UI
     /**
-    * @dev getRewardPerBlock return the current pool reward per block
-    * @param _pid1 the pool id.
-    *        if _pid1 = 0 , it return the current block reward for whole pools.
-    *        otherwise it return the current block reward for _pid1-1.
+    * @dev Return the current pool reward per block
+    * @param _pid The pool id.
```

Comment:
This `getRewardPerBlock` has a very shitty API, as it takes not "the poll Id" but "the pool id plus 1" as its main argument (which is why the argument is named so badly). I created #148 to handle that